modules/ROOT/examples/live-demos/ui-mode/index.js

```diff
@@ -0,0 +1,50 @@
+tinymce.init({
+    selector: 'textarea#iframe',
+    plugins: 'lists help',
+    toolbar: 'undo redo | blocks | bold italic | bullist numlist',
+    ui_mode: 'split',
+    min_height: 500
+  });
+  
+const emailHeaderConfig = {
+  selector: '.tinymce-heading',
+  menubar: false,
+  inline: true,
+  plugins: [
+    'lists',
+    'powerpaste',
+    'autolink'
+  ],
+  toolbar: 'undo redo | bold italic underline',
+  valid_elements: 'strong,em,span[style],a[href]',
+  valid_styles: {
+    '*': 'font-size,font-family,color,text-decoration,text-align'
+  },
+  powerpaste_word_import: 'clean',
+  powerpaste_html_import: 'clean',
+  ui_mode: 'split'
+};
+
+const emailBodyConfig = {
+  selector: '.tinymce-body',
+  menubar: false,
+  inline: true,
+  plugins: [
+    'link', 'lists', 'powerpaste',
+    'autolink', 'tinymcespellchecker'
+  ],
+  toolbar: [
+    'undo redo | bold italic underline | fontfamily fontsize',
+    'forecolor backcolor | alignleft aligncenter alignright alignfull | numlist bullist outdent indent'
+  ],
+  valid_elements: 'p[style],strong,em,span[style],a[href],ul,ol,li',
+  valid_styles: {
+    '*': 'font-size,font-family,color,text-decoration,text-align'
+  },
+  powerpaste_word_import: 'clean',
+  powerpaste_html_import: 'clean',
+  ui_mode: 'split'
+};
+
+tinymce.init(emailHeaderConfig);
+tinymce.init(emailBodyConfig);
```

Comment:
```suggestion
const settings = {
  plugins: 'lists help',
  toolbar: 'undo redo | blocks | bold italic | bullist numlist',
  ui_mode: 'split',
  min_height: 500
};

tinymce.init({
  ...settings,
  selector: 'textarea#iframe'
});

tinymce.init({
  ...settings,
  inline: true,
  selector: '.tinymce-heading',
  inline: true,
  toolbar: 'undo redo | bold italic underline'
});

tinymce.init({
  ...settings,
  selector: '.tinymce-body',
  inline: true
});
```
Wasn't just that one config that needed simplification, so I'll fix it here.